spec/rails_helper.rb

```diff
@@ -163,6 +160,8 @@ def stub_addresses
 
     raise if Partners::Partner.count > 0
     raise if Organization.count > 0
+
+    ENV["RSPEC_HAS_RUN_WEBPACKER"] = "false"
```

Comment:
Maybe just use a class variable here? `@@has_run_webpacker`

Reply:
Definitely more Ruby-centric/idiomatic, but...

```
An error occurred in a `before(:suite)` hook.
Failure/Error: @@has_run_webpacker = false

RuntimeError:
  class variable access from toplevel
# ./spec/rails_helper.rb:164:in `block (2 levels) in <top (required)>'
```

Instance variable `@has_run_webpacker` gets reset each time—`** Running webpacker` gets logged for each `:system` example.

Making it a global variable works—no worse than `ENV` in that sense, I suppose...and not "stringly-typed"